src/Microsoft.Azure.Functions.ExtensionBundle/extensions.json

```diff
@@ -204,6 +204,12 @@
     "name": "MicrosoftDataSqlClient",
     "bindings": []
   },
+  {
+    "id": "Microsoft.AspNetCore.Server.Kestrel.Core",
+    "Version": "2.3.6",

```

Comment:
Why do we need this? Its not added in the preview bundle?

Reply:
Followed up offline